### PR TITLE
Add support for themercury.com.au

### PR DESCRIPTION
**Pull request by Kamleeonau**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,7 @@
 [The Atlantic](https://www.theatlantic.com)\
 [The Australian](https://www.theaustralian.com.au)\
 [The Australian Financial Review](https://www.afr.com)\
+[The Mercury Tasmania](https://www.themercury.com.au)\
```
Comment (iamadamdev) [inline README.md]: Not in alphabetical order

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
 web-ext-artifacts
+/.vs
```
Comment (iamadamdev) [inline .gitignore]: No need to change this

File: options.js [hunk 3]
```diff
@@ -56,6 +56,7 @@ var defaultSites = {
   'The Atlantic': 'theatlantic.com',
   'The Australian': 'theaustralian.com.au',
   'The Australian Financial Review (javascript disabled)': 'afr.com',
+  'The Mercury Tasmania': 'themercury.com.au',
```
Comment (iamadamdev) [inline options.js]: Not in alphabetical order

File: .gitignore (hunk 2 above)
Comment (Kamleeonau) [reply .gitignore]: Whoops... still learning git and using visual studio isn't helping.. sorry about that! Reverted change in new commit.

File: README.md (hunk 1 above)
Comment (Kamleeonau) [reply README.md]: Fixed in new commit

File: options.js (hunk 3 above)
Comment (Kamleeonau) [reply options.js]: Fixed in new commit

**Comment by iamadamdev** [comment]
Fix merge conflicts

**Pull request merged**